.github/workflows/ci.yml

```diff
@@ -414,7 +414,7 @@ jobs:
 
       - name: Run Quarkus Tests in Docker
         run: |
-          mvn clean install -nsu -B -f quarkus/tests/pom.xml -Dkc.quarkus.tests.dist=docker | misc/log/trimmer.sh
+          mvn clean install -nsu -B -f quarkus/tests/pom.xml -Dkc.quarkus.tests.dist=docker -Dtest=StartCommandDistTest | misc/log/trimmer.sh
```

Comment:
🙂  please let us fix this if it's an issue instead of disabling the tests 🙏

Reply:
@andreaTP We are still running one of the main tests for docker. It should cover a lot already, please check the test class.

We can not be blocked at the moment by instability on docker tests. If you have the fix already, please send it and we can revert this change.

Reply:
@andreaTP Moved changes to GHA to this issue https://github.com/keycloak/keycloak/issues/9162.

Reply:
👍